app.py

```diff
@@ -87,6 +98,35 @@ def events(account_id):
     dripData.events.append(request.json["events"][0])
     return Response(None, status=204)
 
+@app.route("/v2/<account_id>/tags", methods=["GET", "POST"])
```

Comment:
In Python coding conventions functions should be separated by 2 empty lines.

```suggestion

@app.route("/v2/<account_id>/tags", methods=["GET", "POST"])
```

Reply:
Added 2 empty lines between functions